build/fake_azure.py

```diff
@@ -83,6 +83,35 @@ def wrapped():
 def test_params() -> 'dict[str, str]':
     return parse_qs(request.headers.get('X-MongoDB-HTTP-TestParams', ''))
 
+@imds.get('/computeMetadata/v1/instance/service-accounts/default/token')
+@handle_asserts
+def get_gcp_token():
```

Comment:
Aside: If we're using this file for multiple mock services, it may be useful to rename it and/or split it into multiple Bottle applications.

Reply:
I'm thinking of just renaming it and not splitting it into multiple Bottle applications, because the mock services at this point are so similar and use identical helper functions.